email-templates/weekly/html.ejs

```diff
@@ -0,0 +1,579 @@
+            </h4>
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <table cellpadding="0" cellspacing="5" class="graph-container">
+              <tbody>
+                <tr>
+
+                  <td class="assets-received-outer-left">
+                    This week your assets received
+                  </td>
+
+                  <td class="assets-received-outer-right">
+                  <% if (weekly.user.topAsset) { %>
+                    Your most popular asset
+                  <% } -%>
+                  </td>
+
+                </tr>
+                <tr>
+                  <td class="assets-received-outer-left">
+
+                    <table cellpadding="0" cellspacing="5" class="assets-received-summary">
+                      <tr>
+                        <td class="assets-received-icon">
+                          <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_comments_large.png') %>" alt="Number of likes" />
+                        </td>
+                        <td class="assets-received-totals">
+                          <div>
+                            <span class="assets-received-count"><%= weekly.user.commentsReceived %></span> comments
+                          </div>
+                          <div class="assets-received-points">
+                            <%= weekly.user.pointsFromCommentsReceived %> points
+                          </div>
+                        </td>
+                      </tr>
+                    </table>
+
+                    <table cellpadding="0" cellspacing="5" class="assets-received-summary">
+                      <tr>
+                        <td class="assets-received-icon">
+                          <img src="<%= baseCollabosphereUrl + resolveAsset('/assets/img/icon_thumbs_up_large.png') %>" alt="Number of likes" />
+                        </td>
+                        <td class="assets-received-totals">
+                          <div>
+                            <span class="assets-received-count"><%= weekly.user.likesReceived %></span> likes
+                          </div>
+                          <div class="assets-received-points">
+                            <%= weekly.user.pointsFromLikesReceived %> points
+                          </div>
+                        </td>
+                      </tr>
+                    </table>
+                  </td>
+
+                  <td class="assets-received-outer-right">
+                  <%
+                  if (weekly.user.topAsset) {
+                    var assetData = getAssetData(weekly.user.topAsset);
+                  -%>
+
+                    <table cellpadding="0" cellspacing="0">
+                      <tbody>
+                        <tr>
+                          <td class="preview-tile-container medium-container" style="background-image: url('<%= assetData.thumbnailUrl %>');" background="<%= assetData.thumbnailUrl %>" valign="bottom">
```

Comment:
@paulkerschen Does `style` have both `background-image:` and `background=` because we're composing HTML for an email? If we were only concerned w/ web browsers I'd think the `background=` is extraneous.